overflow/utils.go

```diff
@@ -9,3 +29,154 @@ func CadenceString(input string) cadence.String {
 	}
 	return value
 }
+
+// HexToAddress converts a hex string to an Address.
+func HexToAddress(h string) (*cadence.Address, error) {
+	trimmed := strings.TrimPrefix(h, "0x")
+	if len(trimmed)%2 == 1 {
+		trimmed = "0" + trimmed
+	}
+	b, err := hex.DecodeString(trimmed)
+	if err != nil {
+		return nil, err
+
+	}
+	address := cadence.BytesToAddress(b)
+	return &address, nil
+}
+
+func parseTime(timeString string, location string) (string, error) {
+	loc, err := time.LoadLocation(location)
+	if err != nil {
+		return "", err
+	}
+	time.Local = loc
+	t, err := dateparse.ParseLocal(timeString)
+	if err != nil {
+		return "", err
+	}
+	return fmt.Sprintf("%d.0", t.Unix()), nil
+}
+
+func getAndUnquoteStringAsPointer(value cadence.Value) *string {
```

Comment:
why return a pointer?

Reply:
Not sure right now but i can check it out :)